datalad_next/create_sibling_webdav.py

```diff
@@ -293,12 +292,15 @@ def _dummy(ds, refds, **kwargs):
         #   'status': 'ok'}]
         for res in ds.foreach_dataset(
             _dummy,
+            recursive=recursive,
+            recursion_limit=recursion_limit,
             return_type='generator',
             result_renderer='disabled',
         ):
             # unwind result generator
             if 'result' in res:
-                yield from res['result']
+                for partial_result in res['result']:
```

Comment:
maybe `res.get('result',[])` then? There is no guarantee that such a field exists.